### PR TITLE
feat-12: Comphrehensive naming consolidation for Base1024Emoji

### LINKED ISSUE #12: Make sure that Base1024Emoji is consistently named "Base1024Emoji"
The variation indicator shall not be omitted.

### PR DESCRIPTION
**Pull request by neruthes**
Close #12.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,34 +11,34 @@ const decoded = decode("Maskbook", Codec.UTF8);
 
 encode(decoded, Codec.Hexadecimal); // '4d61736b626f6f6b'
 encode(decoded, Codec.Base64);      // 'TWFza2Jvb2s='
-encode(decoded, Codec.Base1024);    // '🐟🔂🏁🤖💧🚊😤🀄'
+encode(decoded, Codec.Base1024Emoji);    // '🐟🔂🏁🤖💧🚊😤🀄'
```
Comment (septs) [inline README.md]: keep comments, aligned

File: src/constants/index.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
-import _Base1024Emojis from './emojis.json'
+import _Base1024Emoji_alphabet from './emojis.json'
 
 // constants
-export const base1024Emojis = expandPoints(_Base1024Emojis)
+export const Base1024Emoji_alphabet = expandPoints(_Base1024Emoji_alphabet)
```
Comment (septs) [inline src/constants/index.ts]:
```suggestion
export const Base1024EmojiAlphabet = expandPoints(_Base1024EmojiAlphabet)
```

**Review by septs — CHANGES_REQUESTED** [review]

**Review by Tedko — COMMENTED** [review]
Leave these to developers

File: README.md (hunk 1 above)
Comment (neruthes) [reply README.md]: Done

**Comment by neruthes** [comment]
Are conflicts well resolved? 🤔

cc @septs

**Pull request merged**